unzip.js

```diff
@@ -0,0 +1,61 @@
+(function (exports) {
+    var unzip = require('unzip');
+    var fs = require('fs');
+    var fstream = require('fstream');
+
+    exports.unzipFiles = function unzipFiles() {
+        var fileArray = [
+            './node_modules/predict-pokemon/arff/dataDump_50k_sorted.arff',
```

Comment:
use `__dirname`  e.g. `__dirname + /arff/dataDump_50k_sorted.arff` instead of the static path